tests/integration/flow/test_crud.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+import numpy as np
+import pytest
+
+from jina.flow import Flow
+from jina.proto import jina_pb2
+from jina.types.ndarray.generic import NdArray
+
+@pytest.fixture
+def config(tmpdir):
+    os.environ['JINA_TOPK_DIR'] = str(tmpdir)
+    os.environ['JINA_TOPK'] = '9'
+    yield
+    del os.environ['JINA_TOPK_DIR']
+    del os.environ['JINA_TOPK']
+
+def random_docs(num_docs, embed_dim=10, jitter=1):
+    for j in range(num_docs):
+        d = jina_pb2.DocumentProto()
+        d.tags['id'] = j
+        d.id = str(f'{j}'*16)
+        d.text = b'hello'
+        NdArray(d.embedding).value = np.random.random([embed_dim + np.random.randint(0, jitter)])
+        yield d
+
+def test_delete(config, mocker):
+    NUMBER_OF_SEARCHES = 3
+
+    def validate_results(resp):
+        assert len(resp.search.docs) == NUMBER_OF_SEARCHES
+        for doc in resp.search.docs:
+            assert len(doc.matches) == 0
+
+    response_mock = mocker.Mock(wraps=validate_results)
+
+    docs = list(random_docs(10))
+
+    for method in [
+        'index',
+        'delete'
```

Comment:
We pass the entire document in a `delete`? The API on the Indexers has been to only pass the `id`-s

Reply:
Thanks, I enabled the driver to delete the docs without content as well.